includes/admin/OrderSettings.php

```diff
@@ -495,6 +498,11 @@ private function setInsuranceData(): void
             $insuranceAmount = $this->getCarrierSetting(WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_INSURED_AMOUNT);
         }
 
+        if ($isDefaultInsured && $isBelgium) {
+            $isInsured = $isDefaultInsuredForBE;
```

Comment:
uitlijnen

Reply:
Dit is eigenlijk een hele losse tak van de bovenstaande if statements toch? Dan kan je deze ook aan het begin hangen als de eerste `if`. Nu lijkt het in eerste oogopslag alsof er in deze if nog wat kan gebeuren met een eerder gewijzigde waarde van  `$isInsured` en `$insuranceAmount`

Reply:
Vgm kan je `! $insuranceFromDeliveryOptions && $isBelgium && $isDefaultInsured` doen en dan ook in de volgende 2 takken de checks op `$insuranceFromDeliveryOptions` weglaten